chia/wallet/wallet_node.py

```diff
@@ -225,6 +232,92 @@ async def get_private_key(self, fingerprint: Optional[int]) -> Optional[PrivateK
 
         return key
 
+    def set_resync_on_startup(self, fingerprint, enabled: bool = True):
+        with lock_and_load_config(self.root_path, "config.yaml") as config:
+            if enabled is True:
+                config["wallet"]["reset_sync_for_fingerprint"] = fingerprint
+                self.log.info("Enabled resync for wallet fingerprint: %s", fingerprint)
+            else:
+                self.log.debug(
+                    "Trying to disable resync: %s [%s]", fingerprint, config["wallet"].get("reset_sync_for_fingerprint")
+                )
+                if config["wallet"].get("reset_sync_for_fingerprint") == fingerprint:
+                    del config["wallet"]["reset_sync_for_fingerprint"]
+                    self.log.info("Disabled resync for wallet fingerprint: %s", fingerprint)
+            save_config(self.root_path, "config.yaml", config)
+
+    async def reset_sync_db(self, db_path, fingerprint):
```

Comment:
you don't have any type annotations in this signature (both parameters and return value). I suppose mypy isn't fully enabled on this file. I think it would be good to add it anyway though. The sane goes for `set_resync_on_startup()`. `fingerprint` and return value lack types.